.changeset/cool-deers-chew.md

```diff
@@ -0,0 +1,9 @@
+---
+'@guardian/eslint-plugin-source-foundations': major
+'@guardian/eslint-plugin-source-react-components': major
+'@guardian/source-foundations': major
+'@guardian/source-react-components': major
+'@guardian/source-react-components-development-kitchen': major
+---
+
+Set default textSans line height to 'regular'
```

Comment:
It would be really useful to outline what consumers will need to do to accommodate this

Reply:
The intention is that consumers shouldn't need to make any changes, but we could say that they should expect any textSans copy to now have a tighter line height?